features/FxA-16-devices-view/README.md

```diff
@@ -4,5 +4,19 @@ Device View - Basic UI
 
 https://mozilla.aha.io/features/FXA-16
 
-As A FF user when using Firefox, I want to know the devices and services I am connected to via my Firefox Account, and important information about my the status of my services. For devices, this includes last time synced, record counts of different data types, and optionally data storage size (perhaps an option for self-hosters or others?)
+As a Firefox Accounts user when using Firefox, I want to know the devices and services I am connected to via my Firefox Account, and important information about my the status of my services. For devices, this includes last time synced, and optionally data storage size (perhaps an option for self-hosters or others?)
 
```

Comment:
Completely agree!

I noticed that one when I was copy-pasting the description from Aha, thought "this is exactly the sort of discrepancy that should be noticed when reviewing the PR", and deliberately left it in to see whether it would in fact get noticed ;-)

I would also argue that "data storage size" should be removed, and IIUC we've changed "last time synced" to "last time connected" which is slightly different and easier to implement for the initial version.